jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/dataset/DataSetAwareSelectTest.java

```diff
@@ -40,10 +42,14 @@
 import static org.junit.Assert.*;
 import static org.mockito.Matchers.any;
 import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.doAnswer;
 
 @RunWith(MockitoJUnitRunner.class)
 public class DataSetAwareSelectTest {
 
+    private static final String VALUE_COLUMN_ID = "value";
+    private static final String TEXT_COLUMN_ID = "test";
```

Comment:
Is here "test" put intentionally or it's a typo and should be "text" as in ```TEXT_COLUMN_ID = "text"```?

Reply:
Fixed.